.circleci/config.yml

```diff
@@ -111,6 +111,41 @@ workflows:
             branches:
               only: master
 
+  # Separate workflow just for version tag releases.
+  tagged-release:
+    jobs:
+      - test:
+          filters: &version-tag-only
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*/
+
+      - test_full:
+          filters: *version-tag-only
+
+      - test_full_10:
+          filters: *version-tag-only
+
+      - docker/publish:
+          filters: *version-tag-only
+          requires:
+            - test
+            - test_full
+            - test_full_10
+          image: sourcecred/sourcecred
+          tag: latest
+          extra_build_args: --cache-from=node:12,sourcecred/sourcecred:dev
+          before_build:
+            - pull_cache
+          after_build:
+            - run:
+                name: Publish Docker Tag with Sourcecred Version
+                command: |
+                   DOCKER_TAG=$(docker run sourcecred/sourcecred --version | cut -d' ' -f2)
+                   echo "Version for Docker tag is ${DOCKER_TAG}"
+                   docker tag sourcecred/sourcecred:latest sourcecred/sourcecred:${DOCKER_TAG}
+                   docker push sourcecred/sourcecred
```

Comment:
This is just a personal preference, but for pushes I think it's worth being explicit about the tags being pushed, both for the developer to see as documentation, and in the case that there is any tiny chance that a sourcecred/sourcecred:other or sourcecred/sourcecred:dev container could be pushed by accident.

Reply:
Agree with that :smiley:.
I've made them explicit, but that created an interesting situation.

`docker push sourcecred/sourcecred:${DOCKER_TAG}` would have been enough, because the `:latest` tag would be pushed by the Orb.

But that seems likely to catch someone off guard when they're also seeing an explicit push here. With the https://en.wikipedia.org/wiki/Principle_of_least_astonishment I decided to:
- add `deploy: false` so the Orb doesn't push `:latest`.
- add both push tags explicitly.

What do you think about this?